runtime/lua/vim/lsp/rpc.lua

```diff
@@ -4,6 +4,8 @@ local log = require('vim.lsp.log')
 local protocol = require('vim.lsp.protocol')
 local validate, schedule, schedule_wrap = vim.validate, vim.schedule, vim.schedule_wrap
 
+local is_win = uv.os_uname().version:find('Windows')
```

Comment:
just for reference, a clever way that is common in Lua, that we use elsewhere:

https://github.com/neovim/neovim/blob/63741176ce9bf576ee9693b93844e1cbcf0420b2/test/helpers.lua#L262-L264